apps/web/src/pages/templates/editor/TemplateEditorPage.tsx

```diff
@@ -1,12 +1,63 @@
+import { FormProvider } from 'react-hook-form';
+import { Group, Container } from '@mantine/core';
+import { useParams } from 'react-router-dom';
 import PageContainer from '../../../components/layout/components/PageContainer';
 import PageHeader from '../../../components/layout/components/PageHeader';
 import { TemplatesSideBar } from '../../../components/templates/TemplatesSideBar';
+import { NotificationSettingsForm } from '../../../components/templates/NotificationSettingsForm';
+import { useTemplateController } from '../../../legacy/pages/templates/editor/use-template-controller.hook';
+import { TemplateTriggerModal } from '../../../legacy/components/templates/TemplateTriggerModal';
+import { Button } from '../../../design-system';
 
 export default function TemplateEditorPage() {
+  const { templateId } = useParams<{ templateId: string }>();
+
+  const {
+    selectedMessageType,
+    editMode,
+    template,
+    changeSelectedMessage,
+    isEmbedModalVisible,
+    trigger,
+    isLoading,
+    isUpdateLoading,
+    setValue,
+    onSubmit,
+    loadingEditTemplate,
+    activeChannels,
+    toggleChannel,
+    onTriggerModalDismiss,
+    addMessage,
+    handleSubmit,
+    control,
+    emailMessagesFields,
+    inAppFields,
+    errors,
+    smsFields,
+    methods,
+    removeEmailMessage,
+  } = useTemplateController(templateId);
+
   return (
     <PageContainer>
-      <PageHeader title="Create New Template" />
-      <TemplatesSideBar />
+      <FormProvider {...methods}>
+        <form name="template-form" onSubmit={handleSubmit(onSubmit)}>
+          <PageHeader title="Create New Template" actions={<Button submit>Create</Button>} />
+          <Group align="flex-start" grow>
+            <TemplatesSideBar />
+            <Container mt={15} ml={25} mr={30} fluid padding={0}>
+              <NotificationSettingsForm errors={errors} editMode={editMode} />
+              {trigger && (
+                <TemplateTriggerModal
+                  trigger={trigger}
+                  onDismiss={onTriggerModalDismiss}
+                  isVisible={isEmbedModalVisible}
+                />
+              )}
+            </Container>
+          </Group>
+        </form>
+      </FormProvider>
```

Comment:
![CleanShot 2022-03-07 at 17 38 56](https://user-images.githubusercontent.com/8872447/157066314-292ac32e-4753-4fb4-9814-4cbe222a230e.png)

I think we can stretch the Container to 100% so that the inputs will fill the space. Wdyt?

Reply:
I did that at first, but because we defined maxWidth of all inputs to be 500 it didn't look good.
I can change that